### ISSUE TITLE
the  validation set of audiocaps dataset 

### ISSUE THREAD

**Issue by hello-xiaow** [minguinho26/Prefix_AAC_ICASSP2023#6]
According to the download link you provided, the audiocaps dataset  does not have a validation set. Can you provide audiocaps validation set data？

**Comment by minguinho26** [comment]
Hi, 
I forgot to save the validation set in the .zip file. 

However, you can download the validation set from https://drive.google.com/drive/folders/1e5v-u7qRtmKAzQVMbBSy7CU-tp1nA0su

Please let me know if you have additional questions.

Thank you.

**Comment by blue-blue272** [comment]
> Hi, I forgot to save the validation set in the .zip file.
> 
> However, you can download the validation set from https://drive.google.com/drive/folders/1e5v-u7qRtmKAzQVMbBSy7CU-tp1nA0su
> 
> Please let me know if you have additional questions.
> 
> Thank you.

It seems that the validation set is not right. It is not consistent with the val.csv, can you check it?

**Comment by minguinho26** [comment]
> > Hi, I forgot to save the validation set in the .zip file.
> 
> > 
> 
> > However, you can download the validation set from https://drive.google.com/drive/folders/1e5v-u7qRtmKAzQVMbBSy7CU-tp1nA0su
> 
> > 
> 
> > Please let me know if you have additional questions.
> 
> > 
> 
> > Thank you.
> 
> 
> 
> It seems that the validation set is not right. It is not consistent with the val.csv, can you check it?

I observed same problem when using training set. The reason of mismatching is the deleted youtube videos whose links are contained in val.csv file.
